applications/mobu/values-idfdev.yaml

```diff
@@ -1,15 +1,24 @@
+image:
+  pullPolicy: "Always"
+  tag: "tickets-DM-44635-nicer-github-integration-config"
 config:
   debug: true
-  githubRefreshAppEnabled: true
-  githubCiAppEnabled: true
-  github:
+  githubRefreshApp:
+    accepted_github_orgs:
```

Comment:
If you add that `to_caml` setting, you can now use `acceptedGithubOrgs` here instead, which looks more natural in Helm.